wrappers/python/pysetup.cmake.in

```diff
@@ -1,8 +1,13 @@
 set(ENV{OPENMM_INCLUDE_PATH} "@WRAPPER_INCLUDE_DIRS@")
+set(ENV{CC} "@CMAKE_C_COMPILER@")
+set(ENV{CXX} "@CMAKE_CXX_COMPILER@")
 file(TO_NATIVE_PATH "@CMAKE_INSTALL_PREFIX@/lib" OPENMM_LIB_PATH)
 set(ENV{OPENMM_LIB_PATH} "${OPENMM_LIB_PATH}")
 message("OPENMM_LIB_PATH = " $ENV{OPENMM_LIB_PATH})
 message("OPENMM_INCLUDE_PATH = " $ENV{OPENMM_INCLUDE_PATH})
+message("CC = " $ENV{CC})
+message("CXX = " $ENV{CXX})
```

Comment:
Is this a good thing to do though?  Doesn't Python store the CFLAGS used to build Python itself and then use those to compile extensions?  If this is the case, many of those CFLAGS may be specific to the compilers that built Python in the first place and fail to work when someone specifies a different CC and CXX.

If this is the case, I think just using `python setup.py install` is a sufficient workaround...  But I know I've had problems with this using Conda and MacPorts GCC in the past.  I always need to disable all selected GCC ports before the pyopenmm build will work on my Mac.